src/js/components/DateInput/stories/Range.js

```diff
@@ -15,7 +15,11 @@ export const Range = () => {
   const onChange = (event) => {
     const nextValue = event.value;
     console.log('onChange iso date:', nextValue);
-    console.log('onChange utc date:', new Date(nextValue));
+    console.log(
+      'onChange utc date:',
+      new Date(nextValue[0]),
+      new Date(nextValue[0]),
```

Comment:
Should this be `nextValue[1]`?
```suggestion
      new Date(nextValue[1]),
```